tutorials/intro-express/1-welcome.qmd

```diff
@@ -0,0 +1,310 @@
+---
+title: Getting Started
+---
+
+Shiny for Python is a web application framework that helps tell your
+data story.
+If you've landed on this page,
+you probably have a bit of Python experience,
+worked with data,
+and now need a way to publish an interactive
+web application to help tell your data story.
+
+## Installing Shiny
+
+:::::: {.panel-tabset}
+
+## Windows
+
+::: {.panel-tabset}
+
+## pip
+
+```bash
+python -m venv shiny-env    # create virtual environment
+shiny-env\Scripts\activate  # activate environment
+pip install -U shiny        # install shiny
+```
+
+## conda
+
+```bash
+# create virtual environment and install shiny
+conda create -n shiny-env -c conda-forge shiny
+
+# activate environment
+conda activate shiny-env
+```
+
+## mamba
+
+```bash
+# create virtual environment and install shiny
+mamba create -n shiny-env -c conda-forge shiny
+
+# activate environment
+mamba activate shiny-env
+```
```

Comment:
These make me nervous, in particular because they're completely untested and not something we, as a team, use. If there's a strong guarantee that these will always work, I'd be okay with leaving them in the docs, but if these approaches break or change in the future there's a very small chance that we'd notice and/or update them.

Reply:
I'm less attached to the mamba section. The mamba-only users will know how to modify from the conda install. I do feel we should keep the conda section because of how python data science is taught.

These were some example project installation:

- Stan: https://mc-stan.org/install/
- Scikit learn: https://scikit-learn.org/stable/install.html

This is the current py-shiny conda feedstock: https://github.com/conda-forge/py-shiny-feedstock

I have seen a few projects also put in `uv` install instructions. Since that is a tool the shiny team uses, would that be worth putting in?

Reply:
followup: I'm removing this section so we can move all installation sites to the dedicated installation page